src/PropertiesPanel.js

```diff
@@ -167,6 +172,27 @@ export default function PropertiesPanel(props) {
     getDescriptionForId
   };
 
+  // set-up tooltip context
+  const tooltip = useMemo(() => createTooltipContext(tooltipConfig), [ tooltipConfig ]);
+
+  useEffect(() => {
+    if (typeof tooltipLoaded === 'function') {
+      tooltipLoaded(tooltip);
+    }
+  }, [ tooltip, tooltipLoaded ]);
+
+  const getTooltipForId = (id, element) => {
+    if (id === 'taskDefinitionType') {
```

Comment:
What's going on here? Task definition type is a Zeebe-specific property. I'm surprised to see this in the generic properties panel.

Reply:
Oops, nevermind that 🙈 Lemme fix it

Reply:
I was testing something specific and forgot to remove it. Done now ✅